### PR TITLE
[CLOUD-2418] Update Image name for tech-preview

### PR DESCRIPTION
**Pull request by vbusch**
Signed-off-by: Vanessa Busch <vbusch@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: amq-broker-7-image-streams.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ items:
         version: '1.0'
       from:
         kind: DockerImage
-        name: amq-broker-7/amq-broker-71-openshift:dev
\ No newline at end of file
+        name: amq-broker-7-tech-preview/amq-broker-71-openshift:1.0
```
Comment (dsimansk) [inline amq-broker-7-image-streams.yaml]: @vbusch imho this should point to RH registry
```
registry.access.redhat.com/amq-broker-7-tech-preview/amq-broker-71-openshift:1.0
```

Comment (maschmid) [inline amq-broker-7-image-streams.yaml]: this should also include registry prefix registry.access.redhat.com/

Comment (RoddieKieley) [reply amq-broker-7-image-streams.yaml]: @dsimansk Taking a look at the ose-v1.4.13 tag of jboss-openshift/application-templates I don't see that we have been specific in the past. While I personally prefer to be specific I think being consistent with the overall set of templates that an end developer is going to use is more important.

**Review by RoddieKieley — APPROVED** [review]

File: amq-broker-7-image-streams.yaml (hunk 1 above)
Comment (dsimansk) [reply amq-broker-7-image-streams.yaml]: @RoddieKieley we're in the ImageStream definition of `from`:
https://github.com/jboss-openshift/application-templates/blob/ose-v1.4.13/amq/amq63-image-stream.json#L74-L85

**Pull request merged**